docs/news.rst

```diff
@@ -24,6 +24,7 @@ Psycopg 3.2 (unreleased)
   in non-pipeline mode and not totally reliable (:ticket:`#604`).
   The `Cursor` now only preserves the results set of the last
   `~Cursor.execute()`, consistently with non-pipeline mode.
+- Add support for psycopg[binary] for Python 3.12.
```

Comment:
This will not happen in 3.2, but in the first 3.1.x released after Python 3.12 official release.

Python 3.12 build was tested already in 3.1.10, so `psycopg[c]` already works (unless something changed in pre-release Python). We might even just add packages in 3.1.10.